### ISSUE TITLE
I wonder if the as_loss option has a function to change the format of the output.

### ISSUE THREAD

**Issue by Chigo55** [chaofengc/IQA-PyTorch#276]
I'm training a low-light image enhancement model and I'm going to use ssim and lpips as losses. Does the as_loss option simply have the ability to enable gradient backpropagation? Is there such a function that the output of ssim changes to 1-ssim due to the as_loss option?
